msgraph-mail/go/utilities/models/attachment.go

```diff
@@ -2,6 +2,7 @@ package models
 
 import (
     i336074805fc853987abe6f7fe3ad97a6a6f3077a16391fec744f671a015fbd7e "time"
+    core "github.com/microsoftgraph/msgraph-sdk-go-core"
```

Comment:
can't have a dependency on graph core in the generated outcome, this needs to move to abstractions instead